java/cdk/src/main/java/sleeper/cdk/stack/CompactionStack.java

```diff
@@ -573,7 +570,7 @@ private void lambdaToCreateCompactionTasks(CoreStacks coreStacks,
 
         // Cloudwatch rule to trigger this lambda
         Rule rule = Rule.Builder
-                .create(this, "CompactionMergeTasksCreationPeriodicTrigger")
+                .create(this, "CompactionTasksCreationPeriodicTrigger")
                 .ruleName(SleeperScheduleRule.COMPACTION_TASK_CREATION.buildRuleName(instanceProperties))
                 .description("A rule to periodically trigger the compaction tasks lambda")
```

Comment:
I know this hasn't changed in this PR, but this description would be clearer as "A rule to periodically trigger the compaction tasks creation lambda".